eventrecorder/config.go

```diff
@@ -21,10 +21,10 @@ type (
 		// pgxPoolConfig is instantiated by parsing config.dbDSN.
 		pgxPoolConfig *pgxpool.Config
 	}
-	Option func(*config) error
+	option func(*config) error
```

Comment:
why is this no longer public?

Reply:
Because `config` is not exported either. Without `config` exported there is no way any go file outside the package making use of the option.